### ISSUE TITLE
FPS and resolution settings is gone?

### ISSUE THREAD

**Issue by chong601** [fluddokt/opsu#5]
Title explains it....
Latest version only

**Comment by fluddokt** [comment]
Yeah, I was trying to remove it since the setting doesn't do anything in android.

**Comment by chong601** [comment]
Agreed. But somehow opsu-android actually works fine but the original one have jittery graphics even on 60 FPS

**Comment by itdelatrisu** [comment]
The jitter might be caused by #42, though that's just a guess.  The Android port uses a different music player.

**Comment by chong601** [comment]
I have actually started rewriting the rendering sequence of opsu! (splitting updateAndRender() to Display.update() and use LWJGL Display.sync(int) to take advantage of writing to offscreen buffer and use FPS set on options to timely swap the buffer to the on-screen buffer and also make all animations no longer dependent to FPS) but during my rewrite, I remembered about something like low-latency audio thing that some frameworks do have (not so sure whether LWJGL have them or not). On osu!droid, having low-latency audio stutters the whole game but when it is disabled, everything is fine.

But I still thinking the rendering part caused the jitter but i do take account about the audio issue too, but I will still proceed with the rendering rewrite 1st.

**Comment by chong601** [comment]
Speaking of the jitter, I have actually started rewriting the rendering sequence of opsu! (splitting updateAndRender() to Display.update() which will run at a fixed FPS example 480 fps and use LWJGL Display.sync(int) to take advantage of writing to offscreen buffer and use FPS set on options to timely swap the buffer to the on-screen buffer and also make all animations no longer dependent to FPS). But during my rewrite, I remembered about something like low-latency audio thing that some frameworks do have (not so sure whether LWJGL have them or not). On osu!droid, having low-latency audio stutters the whole game but when it is disabled, everything is fine.
